package/yast2-network.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Jul 12 10:41:58 CEST 2016 - schubi@suse.de
+
+- Added entry "dhclient_set_hostname" to the RNC file.
```

Comment:
As it's a user-oriented changelog, I would replace `RNC file` by `AutoYaST schema` or something like that. What do you think?